### PR TITLE
Add official CoreOS bare metal guide

### PR DESCRIPTION
**Pull request by dghubble**
Link to our bare metal guides and tools from the CoreOS page. @joshix

### CODE REVIEW

**Comment by dghubble** [comment]
I signed it!

File: docs/getting-started-guides/coreos/index.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ Guide and CLI tool for setting up a multi-node cluster on AWS. CloudFormation is
 
 <hr/>
 
+[**Bare Metal Multi-Node**](https://coreos.com/kubernetes/docs/latest/kubernetes-on-baremetal.html)
+
+Guide and HTTP/API service for PXE booting and provisioning a multi-node cluster on bare metal. Ignition is used to provision a master and multiple workers on first boot from disk.
```
Comment (joshix) [inline docs/getting-started-guides/coreos/index.md]: Link to Ignition?

**Comment by joshix** [comment]
One nit

File: docs/getting-started-guides/coreos/index.md [hunk 2]
```diff
@@ -16,6 +16,10 @@ Guide and CLI tool for setting up a multi-node cluster on AWS. CloudFormation is
 
 <hr/>
 
+[**Bare Metal Multi-Node**](https://coreos.com/kubernetes/docs/latest/kubernetes-on-baremetal.html)
```
Comment (joshix) [inline docs/getting-started-guides/coreos/index.md]: Given the context below, do you want to just link straight to https://coreos.com/kubernetes/docs/latest/kubernetes-on-baremetal.html#automated-provisioning ?

**Comment by joshix** [comment]
LGTM

**Comment by johndmulhausen** [comment]
Thanks, @dghubble !

**Pull request merged**